plugins/LossFunction_l1.py

```diff
@@ -0,0 +1,11 @@
+from keras import losses
+
+# L1 loss function = Mean of Absolute Error
+
```

Comment:
Is there any benefit here to having this over just assigning LossFunction to the loss function signature from keras?

LossFunction = losses.mean_absolute_error

Reply:
Wow. After reviewing this code I found it actually doesn't work - it will always use the default loss function as it's not being overwritten in time.

Hold on while I try to figure out how to undo this pull request and fix my code :)